File: backend/components/metiers/threed/api/samples/accessibility/sample_accessibility.js
// sample_accessibility.js – Exemples ultra avancés d'accessibilité pour API Threed (JS)

function sampleAccessibilityCheck(data) {
  // Vérifie l'accessibilité d'une ressource API (clé en main, edge cases inclus)
  if (!data || !data.label) {
    return { accessible: false, reason: 'missing_label' };
  }
  return { accessible: true, lang: data.lang || 'fr' };
}

module.exports = { sampleAccessibilityCheck };

File: backend/components/metiers/threed/api/samples/accessibility/sample_accessibility.py
# sample_accessibility.py – Exemples ultra avancés d'accessibilité pour
# API Threed


def sample_accessibility_check(data):
    """Vérifie l'accessibilité d'une ressource API (clé en main, edge cases
    inclus)."""
    # Logique accessibilité avancée, conforme au cahier des charges
    if not data or "label" not in data:
        return {"accessible": False, "reason": "missing_label"}
    return {"accessible": True, "lang": data.get("lang", "fr")}
